notebooks/04_model_select_and_optimize/hypertune_spark_deep_dive.ipynb

```diff
@@ -28,7 +28,7 @@
   },
```

Comment:
We added in the rest of the notebooks an MS header on the fisrt cell
```
<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>
<i>Licensed under the MIT License.</i>
```
Please note that you have to add `<i></i>` in both lines, if not, it is not rendered correctly on Github.